src/ansys/dpf/core/plotter.py

```diff
@@ -149,9 +156,20 @@ def add_mesh(self, meshed_region, deform_by=None, scale_factor=1.0, **kwargs):
         grid.set_active_scalars(None)
         self._plotter.add_mesh(grid, **kwargs_in)
 
-    def add_point_labels(self, nodes, meshed_region, labels=None, **kwargs):
+    def add_point_labels(
+        self,
+        nodes: Union[Nodes, List[Node], List[int]],
+        meshed_region: MeshedRegion,
+        labels: Union[List[str], None] = None,
+        **kwargs,
+    ) -> List:
         label_actors = []
-        node_indexes = [meshed_region.nodes.mapping_id_to_index.get(node.id) for node in nodes]
+        if isinstance(nodes, Nodes):
```

Comment:
You could maybe add another test with `nodes` being a `Nodes` and silence this Codecov warning